src/padmaja/PatternPrinting.java

```diff
@@ -0,0 +1,41 @@
+package padmaja;
+
+public class PatternPrinting {
+	void printStartPattern() {
+		for (int i = 1; i <= 4; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible